inductiva/coastal/README.md

```diff
@@ -8,83 +8,61 @@ is performed using the [SWASH](https://swash.sourceforge.io/) simulator.
 
 ### Example
 
+Note: You can follow this example in a notebook or run the script `test.py`
```

Comment:
change the name test.py to something that is much more explicit. For example random_bathymetry_demo.py (if that is the case)